tests/benchmarks/micro_benchmarks/test_cuda_nccl_bw_performance.py

```diff
@@ -109,6 +110,9 @@ def test_nccl_bw_performance(self, allgather, allreduce, reduce, broadcast, redu
         assert (benchmark.result['alltoall_8589934592_algbw'][0] == 256.36)
         assert (benchmark.result['alltoall_8589934592_busbw'][0] == 224.31)
 
+        # Switch to in-place
```

Comment:
why need to change the original test, i think we'd better add a new testcase for in place?